apps/frontend/src/routes/blog.$blogId.lazy.tsx

```
@@ -0,0 +1,79 @@
+
+    const {
+        data: post,
+        isLoading,
+        error,
+    } = trpc.getPost.useQuery(
+        {
+            id: Number(blogId),
+            withRelations: true,
+        },
+        {
+            enabled: !isNaN(Number(blogId)),
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)